NEWS.md

```diff
@@ -0,0 +1,42 @@
+# Changes by Version
+
+## Unreleased
```

Comment:
I thought the Unreleased section should only appear if it actually has a body. Otherwise, it's not shown. This is similar to any of the other tags, they're only used when there's info for them.
Otherwise, the way it's shown right now, it feels that v0.7.1 is an unreleased version, which is not true.

Reply:
Well, the top example in the "keep a changelog" spec has an empty "Unreleased" section, and it's the same heading level as the other versions.

Reply:
That's true, on text it's obvious that **Unreleased** and **v0.7.1** are a different section, but once the markdown is processed the difference is not obvious. I think that "keep a changelog" adds square brackets to make it more obvious and keep a consistent formatting.

Reply:
OK, I'll change it so that "Unreleased" links to `compare/v0.7.1...master`.